web/index.css

```diff
@@ -416,3 +380,31 @@ div.toggle {
   padding: 4px;
   background: #3f3f3f;
 }
+
+.sharing-dialog {
+  top: 100px;
+  left: 100px;
+  right: 100px;
+}
```

Comment:
The sharing-dialog feels a bit too wide for me.
Making it 500-600px pixels wide feels better to me:

```
.sharing-dialog {
  top: 100px;
  left: 0;
  right: 0;
  margin: 0 auto;
  width: 600px;
}
```

Or maybe we could use something like this as the default for any dialog ?